.github/workflows/generate-badges.yml

```
@@ -0,0 +1,43 @@
+        paths-ignore: 
+            - coverage-badge.svg
+            - tests-badge.svg
+
+permissions:
+    contents: write
+
+jobs:
+    generate_badge:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Checkout
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 8

(indentation)